src/Digipolis.Caching/Options/CacheSettings.cs

```diff
@@ -19,7 +18,7 @@ internal class CacheSettings : SettingsBase
 
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "HAA0302:Display class allocation to capture closure", Justification = "<Pending>")]
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "HAA0301:Closure Allocation Source", Justification = "<Pending>")]
-        public static void RegisterConfiguration(IServiceCollection services, IConfigurationSection section, IHostingEnvironment env)
+        public static void RegisterConfiguration(IServiceCollection services, IConfigurationSection section, IWebHostEnvironment env)
```

Comment:
mogelijk beter generieke IHostEnvironment gebruiken https://andrewlock.net/ihostingenvironment-vs-ihost-environment-obsolete-types-in-net-core-3/

Reply:
Ooh wacht, IHostingEnvironment is deprecated.
Ik zie om het aan te passen.